entries/b/blockfi.com.json

```diff
@@ -5,6 +5,7 @@
       "totp"
     ],
     "documentation": "https://help.blockfi.com/hc/en-us/articles/360049343611",
+    "recovery": "https://help.blockfi.com/hc/en-us/articles/360048862652",
```

Comment:
I'm getting a page that says that their help center no longer exists.

Reply:
The same for me, but the page was available when I opened the PR. For now, I will remove both the doc and rec URLs.